Makefile

```diff
@@ -111,6 +111,8 @@ endef
 crd: ensure-controller-gen ensure-yq
 	rm -rf ./config/crds
 	(cd apis; '../$(CONTROLLER_GEN)' crd:crdVersions=v1 paths=./hive/v1 paths=./hiveinternal/v1alpha1 output:dir=../config/crds)
+	# Generate CMM crds
+	$(CONTROLLER_GEN) crd paths=./vendor/sigs.k8s.io/cluster-api/... paths=./thirdparty/... output:crd:dir=./config/crds
```

Comment:
Noyce.

Do we need `:crdVersions=v1` or is that the default at this point? (If so, can we (separately) remove it from L113?)